markusmoss/cli.py

```diff
@@ -45,6 +45,8 @@ def _parse_args():
     parser.add_argument("--generate-config", nargs='?', default=-1)
     parser.add_argument("-f", "--force", action="store_true")
     parser.add_argument("-v", "--verbose", action="store_true")
+    parser.add_argument("-s", "--select", nargs='+',
```

Comment:
Overall I think it's a good idea to keep the CLI argument names identical to the TOML config values (except hyphens vs. underscores). I think `--selected-groups` is better than `--select` here.